samples/cellular/nrf_cloud_multi_service/README.rst

```diff
@@ -1420,6 +1420,52 @@ To onboard devices manually, you can use the `Bulk Onboard Devices <nRF Cloud Po
 
 If you were directed here as part of other instructions, proceed to the next step of those instructions.
 
+Running on native_sim
+=====================
+
+This sample can be run on the `native_sim` target. That way, you can try out connectivity without the need for embedded hardware.
+A Linux host or docker container is required to run the `native_sim` target.
+Some setup is needed in order to connect the application to the network. See the :ref:`networking_with_native_sim` for more information. Here, the `nat.conf` configuration is used.
+Refer to :ref:`nrf_cloud_multi_service_create_device_cred_locally` for creating the necessary device credentials locally.
+Then, modify the `overlay_native_sim.conf` file to use the client id for the credentials you created.
+
+   .. code-block:: console
+
+      # build nrf_cloud_multi_service with native_sim configs, optionally use -DEXTRA_CONF_FILE="overlay_native_sim.conf;overlay_native_sim_coap.conf" for CoAP
+      west build -b native_sim -- -DEXTRA_CONF_FILE="overlay_native_sim.conf" -DSB_CONF_FILE="sysbuild_native_sim.conf"
+      # set up zephyr ethernet interface
+      ~/work/ncs/tools/net-tools$ sudo ./net-setup.sh --config nat.conf start
+      # run the application (use CTRL-C to exit)
+      ./build/nrf_cloud_multi_service/zephyr/zephyr.exe
+      # clean up zephyr ethernet interface
+      ~/work/ncs/tools/net-tools$ sudo ./net-setup.sh --config nat.conf stop
+
+If you intend to trace network traffic on the `zeth` interface, it can be useful to modify the `ssl_tls12_populate_transform` function inside `mbedtls` close to `MBEDTLS_SSL_KEY_EXPORT_TLS12_MASTER_SECRET` to print the TLS ephemeral keys:
```

Comment:
```suggestion
If you intend to trace network traffic on the ``zeth`` interface, it can be useful to modify the :c:func:`ssl_tls12_populate_transform` function inside ``mbedtls`` close to ``MBEDTLS_SSL_KEY_EXPORT_TLS12_MASTER_SECRET`` to print the TLS ephemeral keys:
```